### ISSUE TITLE
govendor keeps adding spurious `context` dependency

### ISSUE THREAD

**Issue by diwakergupta** [kardianos/govendor#200]
Hello, I'm using govendor for [dbxcli](https://github.com/dropbox/dbxcli). Everytime I run `govendor add` or `govendor fetch`, it keeps adding a spurious `context` entry in my vendor.json:

```diff
--- vendor/vendor.json
+++ vendor/vendor.json
@@ -3,64 +3,68 @@
        "ignore": "appengine test",
        "package": [
                {
-                       "checksumSHA1": "/VtWKyLBaqn7b3yWJi8X/TmpowA=",
+                       "path": "context",
+                       "revision": ""
+               },
```

When I run `govendor list`, it shows the context package "missing":

```
$ govendor list
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/async
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/auth
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/files
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/properties
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/sharing
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/team
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/team_common
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/team_policies
 v  github.com/dropbox/dropbox-sdk-go-unofficial/dropbox/users
 v  github.com/dustin/go-humanize
 v  github.com/grantseltzer/golumns
 v  github.com/inconshreveable/mousetrap
 v  github.com/mitchellh/go-homedir
 v  github.com/mitchellh/ioprogress
 v  github.com/spf13/cobra
 v  github.com/spf13/pflag
 v  golang.org/x/crypto/ssh/terminal
 v  golang.org/x/net/context
 v  golang.org/x/oauth2
 v  golang.org/x/oauth2/internal
pl  github.com/dropbox/dbxcli
 l  github.com/dropbox/dbxcli/cmd
  m context
```

I've tried manually removing the context entry many times, but govendor keeps adding it back. It is a minor nuisance, but also quite confusing, especially for new contributors to the project.

Is this a known issue? Any workaround until there's a real fix?

**Comment by kardianos** [comment]
What command do you use? +outside?
If you haven't updated to go1.7, it will show it. There is another outstanding issue to teach govendor a std pkg list rather then inspecting GOROOT.

One solution would be to add `context/` to the ignore list in the vendor.json file.

**Comment by diwakergupta** [comment]
@kardianos I just use `govendor list` without any args. What's the dependency on go1.7? I'm using go1.6.3. Note that `golang.org/x/net/context` is already vendored in the project and `govendor` recognizes that.

Can you summarize the root cause? I'm having trouble understanding what the problem actually is. What is this other `context` package and where is it coming from?

**Comment by kardianos** [comment]
The latest `x/net/context` references the new `go1.7` package `context` behind a build flag. Which is probably where you are getting that from.

You could ignore "go1.7" or you could ignore "context/" for now, either one should make it go away until you upgrade.

**Comment by diwakergupta** [comment]
Perfect, thanks!
